hack/examples/README.md

```diff
@@ -2,6 +2,7 @@
 
 To help you make the most of Nuclio, the following function examples are provided:
 
+Note: all function examples have the explicit field `disableDefaultHttpTrigger: false`, so they are deployable even if default http trigger creation is disabled on platform configuration level.
```

Comment:
need to add reference to `docs/reference/function-configuration/function-configuration-reference.md`

Reply:
@rokatyy - Perhaps I was not clear enough.
What I meant is that you should add information as well on `docs/reference/function-configuration/function-configuration-reference.md` to make sure this file explains the new configuration as well.

Reply:
@liranbg added :)